pkg/acsengine/testdata/v20160330/dcos.json

```diff
@@ -12,13 +12,9 @@
       {
         "name": "agentprivate",
         "count": 3,
-        "vmSize": "Standard_D2_v2"
```

Comment:
Modified agent pool data to have only one pool, as per the normal spec for this version.